backend-tests/tests/test_rbac_v2.py

```diff
@@ -36,21 +36,32 @@
 from testutils.api.client import ApiClient
 
 
-def login(user):
+def login(user, use_personal_access_token=False):
     """
     login authenticates the user and saves the user token
     with user object
+    in case of use_personal_access_token===True it saves the personal access token
     """
     useradm_MGMT = ApiClient(useradm.URL_MGMT)
     rsp = useradm_MGMT.call("POST", useradm.URL_LOGIN, auth=(user.name, user.pwd))
     assert rsp.status_code == 200, "Failed to setup test environment"
     user.token = rsp.text
 
+    if use_personal_access_token:
```

Comment:
If we want to use personal access token, I'd move this part to the beginning of function, like this:
```
if use_personal_access_token:
   . . . 
else:
    useradm_MGMT = ApiClient(useradm.URL_MGMT)
    rsp = useradm_MGMT.call("POST", useradm.URL_LOGIN, auth=(user.name, user.pwd))
    assert rsp.status_code == 200, "Failed to setup test environment"

user.token = rsp.text
```

Do we want to execute this part of code even if PAT is being used?
```   useradm_MGMT = ApiClient(useradm.URL_MGMT)
    rsp = useradm_MGMT.call("POST", useradm.URL_LOGIN, auth=(user.name, user.pwd))
    assert rsp.status_code == 200, "Failed to setup test environment"
    user.token = rsp.text
```

Reply:
yes, we do. we need to be logged in to issue a token.

Reply:
Excuse me, my bad! I'll raise another concern in different comment

Reply:
Please at least update the docstring with precise information. Thanks!

Reply:
I was so happy I saw all green here, that I just pressed merge. updated the comment here: https://github.com/mendersoftware/integration/pull/2064